GameModes/OnlineGameMode.cs

```diff
@@ -149,5 +165,47 @@ public virtual void LobbyReadyCheck()
         { 
             
         }
+
+        internal virtual void PlayerLeftLobby(OnlinePlayer player)
+        {
+            
+        }
+
+        internal virtual void NewPlayerInLobby(OnlinePlayer player)
+        {
+
+        }
+
+        internal virtual void LobbyTick(uint tick)
```

Comment:
If we are running at 20 frames and incrementing a ticket every frame (as I'm seeing in OnlineManager), how often are we updating the player cached data? Is it based on cache size, staleness of data, or number of ticks?